src/modules/producer.js

```diff
@@ -115,29 +144,30 @@ function checkRpc (queue, msg, options) {
 function produce(queue, msg, options) {
   //default options are persistent and durable because we do not want to miss any outgoing message
   //unless user specify it
-  options = Object.assign({ persistent: true, durable: true }, options);
-
+  options = Object.assign({persistent: true, durable: true}, options);
+  
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.dialonce.io/coding_rules#rule_key=javascript%3ATrailingWhitespace)